src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et3ResponseService.java

```diff
@@ -137,10 +142,31 @@ public void saveRelatedDocumentsToDocumentCollection(CaseData caseData) {
      * @param caseDetails Contains details about the case.
      */
     public void sendNotifications(CaseDetails caseDetails) {
-        emailService.sendEmail(et3EmailTribunalTemplateId,
+        emailService.sendEmail(
+            et3EmailTribunalTemplateId,
             caseDetails.getCaseData().getTribunalCorrespondenceEmail(),
             buildPersonalisation(caseDetails)
         );
     }
 
+    // todo group all personalisation methods and NotificationServiceConstants to EmailService or to a
+    // PersonalisationService. Might even be able to join into one single method that prepares all the data for all
+    // emails (unless some of it is expensive / bespoke / overrides in which case send a map parameter of overrides)
+    private Map<String, String> buildPersonalisation(CaseDetails caseDetails) {
+        CaseData caseData = caseDetails.getCaseData();
+        Map<String, String> personalisation = new ConcurrentHashMap<>();
+        personalisation.put("case_number", caseData.getEthosCaseReference());
+        personalisation.put(CLAIMANT, caseData.getClaimant());
+        personalisation.put("list_of_respondents", getRespondentNames(caseData));
+        personalisation.put(DATE, ReferralHelper.getNearestHearingToReferral(caseData, "Not set"));
+        personalisation.put(LINK_TO_EXUI, emailService.getExuiCaseLink(caseDetails.getCaseId()));
+        return personalisation;
+    }
+
+    // todo probably should be replaced with Helper.getRespondentNames
```

Comment:
Still relevant?

Reply:
Yeah, just not relevant to this ticket, it's quite a big thing the personalisation refactor that should happen here